packages/theme-material-ui/scripts/buildThemes.js

```diff
@@ -0,0 +1,272 @@
+/*
+  Build themes
+  ============
+
+  Write themes in JSON format to the `build/json` directory.
+  Each theme goes into a directory named like this: `build/json/colorSchemeDensityTheme`.
+  Each theme directory contains four files: `resolvedRoles.json`,
+  `unresolvedRoles.json`, `metadata.json`, and `theme.json`.
+  `theme.json` contains metadata and resolved roles.
+
+*/
+
+import path from "path";
+import fs from "fs";
+import mkdirp from "mkdirp";
+import dedent from "dedent";
+import colorSchemes from "../../theme-data/src/colorSchemes";
```

Comment:
we should reference the `@hig/theme-data` package rather than using the relative paths from the repo. there can be instances where people may download this package only and the relative paths will not work.